src/plugins/visualizations/public/components/visualization_error.tsx

```diff
@@ -22,7 +22,7 @@ export class VisualizationError extends React.Component<VisualizationErrorProps>
         iconColor="danger"
         data-test-subj="visualization-error"
         body={
-          <EuiText size="xs">
+          <EuiText size="xs" className="visualization-error-text">
```

Comment:
Where is this class name coming from? I don't see it defined anywhere

Reply:
Looks like it was added for functional tests to remove the EUI targeted class in this file: https://github.com/elastic/kibana/pull/133927/files/2d13ef99c04e416e3dde4b8ee8a6624c8b6f0fa5#diff-31d8c0a9483e4698315d644b2696ced4a972789e17f52886a3d8f43af1acf78c

Reply:
I see, makes sense. I guess a data-test-subj would be better suited, but not a blocker.

Reply:
As the prompt has the test subject `visualization-error`, I think `visualization-error-text` or `visualization-error-body` is suitable

Reply:
Updated in [a5c8b08](https://github.com/elastic/kibana/pull/133927/commits/a5c8b08e4029ede6019bb31f8639131a1c1d42f4)